examples/drupal_loadtest.rs

```diff
@@ -22,48 +22,48 @@
 //! See the License for the specific language governing permissions and
 //! limitations under the License.
 
+use goose::boilerplate;
+boilerplate!();
```

Comment:
@alsuren What do you think this is just confusing? Or is there a cleaner way to do this? It's just a big block of `use` that has to go at the start of every load test: I figure as long as I document this people can choose which they prefer, this macro or manually declaring everything.

Reply:
removed nonstandard `boilerplate!()` macro